src/cardanoWallet.ts

```diff
@@ -31,7 +31,7 @@ import { DirPath } from './common';
  *   - `'direct'` - enables fetching stake pool metadata from
  *     the registered pool URL.
  *   - otherwise, the URL of a SMASH metadata proxy server
- *     can be supplied.
+ *     can be supplied. This must not contain the endpoint path.
```

Comment:
Probably no point in validating it here in the code, or should we?

Reply:
Right, just have this option match the cardano-wallet CLI, and cardano-wallet can validate it if needed.